requirements/edx/openstack.txt

```diff
@@ -7,4 +7,4 @@
 setuptools==24.0.3
 
 # OpenStack swift backend for django storage API, with lazy connection loading
-git+https://github.com/open-craft/django-storage-swift.git@jill/lazy_connect#egg=django-storage-swift==1.2.15
+git+https://github.com/open-craft/django-storage-swift.git@opencraft-release/ginkgo.1#egg=django-storage-swift==1.2.15
```

Comment:
Can't we move to the new 1.2.18 release now as well?

Reply:
Oh of course!  Fixed with 193261ee5626e91e5152aff41e48ac0040cab9ce.